can_db/e46.md

```diff
@@ -4,24 +4,30 @@ The CAN bus is not available at the OBD-II port, so it must be tapped into behin
 The easiest places to do this are either behind the intrument cluster or at the steering angle sensor, 
 which is located at the base of the steering column near the firewall.
 
+![Steering Angle Sensor Plug](../images/e46_sensor_plug.jpg)
+
 A simple plug-in harness can be made by cutting both sides of this connector and a few inches of wiring 
-from a donor car at the salvage yard.  Constant and switched power can also be found at this connector.
+from a donor car at the salvage yard.  The CAN-H and CAN-L are the Yellow/Black and Yellow/Brown wires, respectively.
+Constant/switched power and ground can also be found at this connector.  The harness pictured does not include the 
+power/ground.  
+
+![CAN bus harness](../images/e46_harness.jpg)
 
 ## Recommended CAN IDs:
 
 Channel name | CAN ID | Equation | Notes
 ------------ | --- | -------- | -----
-Accelerator position (%) | 809 | `F * 100 / 255` |
+Accelerator position (%) | 809 | `F / 2.54` |
 Brake position (%) | 809 | `G & 1 * 100` | Brake pressures are not available on the main CAN bus, so this is just an on/off signal from the brake switch.
-Steering angle | 501 | `if(b & 0x80, 0 - ((b & 0x7f)<<8) \| a, (b<<8) \| a) * 0.04394` | Positive value = turning right. You can swap the equation in the if statement if you prefer it the other way around.
+Steering angle | 501 | `if(B & 0x80, -1, 1) * bitsToUIntLe(raw, 0, 15) * 0.04394` | Positive value = turning right. You can swap the equation in the if statement if you prefer it the other way around.
```

Comment:
FYI I ended up flipping positive/negative in the BRZ/86 equations to match how RaceChrono visualizes the steering angle. You may also want to do left=positive.